osgp/platform/osgp-adapter-ws-publiclighting/src/main/java/org/opensmartgridplatform/adapter/ws/publiclighting/endpoints/PublicLightingAdHocManagementEndpoint.java

```diff
@@ -255,6 +256,26 @@ public GetStatusResponse getGetStatusResponse(@OrganisationIdentification final
         return response;
     }
 
+    private org.opensmartgridplatform.adapter.ws.schema.publiclighting.adhocmanagement.DeviceStatus getDeviceStatus(
+            final ResponseMessage message) {
+        DeviceStatus deviceStatus;
+        if (message.getDataObject() instanceof LightSensorStatus) {
+            deviceStatus = this.createDeviceStatusFromLightSensorStatus((LightSensorStatus) message.getDataObject());
+        } else {
+            deviceStatus = (DeviceStatus) message.getDataObject();
+        }
+        return this.adHocManagementMapper.map(deviceStatus,
+                org.opensmartgridplatform.adapter.ws.schema.publiclighting.adhocmanagement.DeviceStatus.class);
+    }
+
+    private DeviceStatus createDeviceStatusFromLightSensorStatus(final LightSensorStatus lightSensorStatus) {
+        final List<LightValue> sensorValues = new ArrayList<>();
+        sensorValues.add(new LightValue(1, lightSensorStatus.isOn(), 100));
```

Comment:
The dim value does not make a lot of sense for the light sensor status. If setting it to null causes issues, some default is probably pragmatically OK for now, and 100 seems as good as any other. Something similar applies to the defaults below in the `DeviceStatus` constructor.
Thorough fixing of modelling issues with light values and device statuses looks definitely out of scope for what was considered in making get status work by fixing correlation UID issues.